vm-setup/setup/nanorc

```diff
@@ -0,0 +1,26 @@
+
+# The following is appended to /etc/nanorc during the VM setup.
+# Leave one empty line at the top and at the bottom.
```

Comment:
where's the empty line at the bottom?

Reply:
Unknown. I did not modify this file during the consolidation of the microservice directories.